spec/test_ansible_lint.py

```diff
@@ -2,10 +2,10 @@
 def test_ansible_lint_is_installed_at_version_5_0_12_(host):
```

Comment:
Also here, the test method names still refer to the old 5_0_12 version. 

Note: they will be rendered in the test output so that it reads like a specification:

![image](https://user-images.githubusercontent.com/365744/151452995-da164357-13ad-494e-b0c2-a06217ee3679.png)

Reply:
I agree, how could I forget about this :D Thank you for pointing it out.